manager/tunnel_unix.go

```diff
@@ -114,5 +115,6 @@ func StopAll() {
 	last := GetLastKnownState()
 	for _, tunnel := range last.Tunnels {
 		Stop(tunnel.PID)
+		// add a stop via REST API func here.  So rest api gets signal as well.
```

Comment:
Is this needed? If we're sending a SIGINT to the SC client process, the client itself should be sending that already.